migrations/1629217630132-encrypt_encoded_mfa_settings.ts

```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from 'typeorm'
+import { SnCryptoNode } from '@standardnotes/sncrypto-node'
+
+export class encryptEncodedMfaSettings1629217630132 implements MigrationInterface {
```

Comment:
Why we put random(?) numbers after class name?

Reply:
They are timestamps :)

Reply:
I see, but why? :) I believe it's not done to disguise class names, but can't understand the reasoning behind it.

Reply:
Common in migrations, to order them. Otherwise you have a bunch of unordered classes.

Reply:
Ah, got it, thanks :)